orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/servergroup/ServerGroupCacheForceRefreshTask.groovy

```diff
@@ -115,6 +119,7 @@ class ServerGroupCacheForceRefreshTask extends AbstractCloudProviderAwareTask im
     boolean allUpdatesApplied = false
     refreshableServerGroups.each { Map<String, String> model ->
       try {
+        log.info("Force cache refresh POST a cache refresh request to clouddriver with model: ${model}")
```

Comment:
I don't really understand this log message. What's the intent here? I'm happy to help massage it.

Reply:
This is mostly to indicate that orca posted a new FCR to clouddriver. Helpful to see what's being sent to clouddriver.